shared/cfg/guest-os/Linux/RHEL.cfg

```diff
@@ -5,7 +5,7 @@
         modprobe_module = acpiphp
     block_hotplug:
         modprobe_module = acpiphp
-    unattended_install, check_block_size, svirt_install, with_installation:
+    unattended_install, check_block_size..extra_cdrom_ks,svirt_install, with_installation:
```

Comment:
I think check_block_size has cover your case check_block_size..extra_cdrom_ks. this change is necessary

Reply:
check_block_size have two scenarios, some cases are not related to installation, it have side-effect if not distinguish them